src/pages/publication/references_tab/components/IsbnListField.tsx

```diff
@@ -18,17 +19,18 @@ const IsbnListField: FC = () => {
       {({ field, form: { setFieldValue, setFieldTouched }, meta: { error } }: FieldProps) => (
         <Autocomplete
           freeSolo
+          autoSelect
           multiple
           options={[]}
           value={field.value ?? ''}
           onChange={(_: ChangeEvent<{}>, value: string[], reason) => {
             setFieldTouched(field.name);
-            if (reason === 'create-option') {
+            if (['create-option', 'blur'].includes(reason)) {
```

Comment:
hva betyr dette? 🤔

Reply:
Vi vil tolke både `blur` og `Enter` som en "legg til ISBN"-handling. Autocomplete sender ulik `reason` for de to.
Ellers slipper jeg å skrive `reason` hele 2 (!!!) ganger med å bruke includes :D 
`['create-option', 'blur'].includes(reason)`
=
`reason === 'blur' || reason === 'create-option'`
syns du det var mer forvirrende?

Reply:
haha, synes bruken av includes var mer forvirrende enn alternativet ja 😝